CMakeLists.txt

```diff
@@ -51,7 +51,9 @@ if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
 endif()
 
 # Install in project directory
-set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+	set(CMAKE_INSTALL_PREFIX ${PROJECT_SOURCE_DIR})
+endif()
 # Extra CMake modules (Find*.cmake) are in CMakeModules/
```

Comment:
Personally I'd vote for not setting it at all. I'm happy to leave the prefix as default and instead set a DESTDIR value in `build.sh`.

I think it causes more issues than it fixes when you are trying to second guess what the user wants when they leave he prefix as default, to me overriding even the default is a bad idea.